browser/src/VariantPage/GnomadPopulationsTable.tsx

```diff
@@ -102,14 +103,24 @@ export class GnomadPopulationsTable extends Component<
   }
 
   render() {
-    const { datasetId, exomePopulations, genomePopulations, showHemizygotes, showHomozygotes } =
-      this.props
+    const {
+      datasetId,
+      exomePopulations,
+      genomePopulations,
+      jointPopulations,
+      showHemizygotes,
+      showHomozygotes,
+    } = this.props
     const { includeExomes, includeGenomes } = this.state
 
     const mergedPopulations = mergeExomeGenomeAndJointPopulationData({
       exomePopulations: includeExomes ? exomePopulations : [],
       genomePopulations: includeGenomes ? genomePopulations : [],
-    })
+      jointPopulations,
+      // @ts-expect-error: silence TS's warning -- there is no legitimate "" id in
+      //   PopulationId, but the source array from joint population has a summary value
+      //   with an id of "", filter it out here to display the correct values
+    }).filter((mergedAncestry) => mergedAncestry.id !== '')
```

Comment:
You might be able to avoid adding the `ts-expect-error` by adding an explicit cast: `filter((mergedAncestry) => (mergedAncestry.id as string) !== '')`. Definitely the lesser evil here if it works.